dds/DCPS/GuidUtils.h

```diff
@@ -102,6 +102,13 @@ struct OpenDDS_Dcps_Export GUID_tKeyLessThan {
 
 typedef OPENDDS_SET_CMP(RepoId, GUID_tKeyLessThan) RepoIdSet;
 
+struct SeqReaders {
```

Comment:
This doesn't belong in GuidUtils.h since it introduces a dependence on sequence numbers (and the idea of readers / writers). I'd just put it in RtpsUdpDataLink.h for now.